lib/basis_transcode.cpp

```diff
@@ -358,10 +358,14 @@ ktxTexture2_transcodeUastc(ktxTexture2* This,
         memcpy(priv._levelIndex, protoPriv._levelIndex,
                This->numLevels * sizeof(ktxLevelIndexEntry));
         // Move the DFD and data from the prototype to This.
-        delete This->pDfd;
+        if (This->pDfd) {
+            free(This->pDfd);
+        }
         This->pDfd = prototype->pDfd;
         prototype->pDfd = 0;
-        delete This->pData;
+        if (This->pData) {
```

Comment:
Please remove this `if` as well for the same reasons as given above.

Reply:
done.